ports/mimxrt10xx/common-hal/microcontroller/__init__.c

```diff
@@ -72,10 +72,10 @@ void common_hal_mcu_on_next_reset(mcu_runmode_t runmode) {
         }
         // Pretend to be the first of the two reset presses needed to enter the
         // bootloader. That way one reset will end in the bootloader.
-        _bootloader_dbl_tap = DBL_TAP_MAGIC;
+        SNVS->LPGPR[0] = DBL_TAP_MAGIC;
```

Comment:
Have you verified that this register survives a reset with a reset button?

Reply:
The datasheet implies that it does but I haven't confirmed it:

![Screen Shot 2020-01-21 at 10 26 47 AM](https://user-images.githubusercontent.com/52649/72860851-01fa9780-3c7d-11ea-8e52-6e2fdb509b5d.png)

We can't use RAM for this because we shuffle the underlying RAM banks on start up which moves the value that the UF2 bootloader sets to somewhere else.